src/oncall/user_sync/ldap_sync.py

```diff
@@ -382,12 +384,19 @@ def sync(config, engine):
     for username in users_to_purge:
         prune_user(engine, username)
 
+    # mark active and re-add contact information
     logger.debug('Users to reactivate:')
     for username in users_to_reactivate:
         logger.debug(username)
         try:
             engine.execute('UPDATE user SET active = TRUE WHERE name = %s', username)
             stats['users_reactivated'] += 1
+            for key, value in ldap_users[username].items():
+                if value and key in modes:
+                    logger.debug('\t%s -> %s' % (key, value))
```

Comment:
If we need this log statement, the logging module will do the string interpolation for you. You can replace this with `logger.debug('\t%s -> %s', key, value)`